doc/sonic-swss-common/ZMQ producer-consumer state table design.md

```diff
@@ -0,0 +1,60 @@
+# ZMQ producer/consumer state table design
+
+# Table of Contents
+- [Table of Contents](#table-of-contents)
+- [1 Functional Requirements](#1-functional-requirement)
+  * [1.1 Supported operations](#11-supported-operations)
+  * [1.2 ZMQ producer state table support async operation](#12-zmq-producer-state-table-support-async-operation)
+  * [1.3 ZMQ consumer state table support async operation](#13-zmq-consumer-state-table-support-async-operation)(#2-configuration-and-management-requirements)
+- [2 Design](#1-design)
+  * [2.1 diagram](#21-diagram)
+  * [2.1 flow chart](#21-flow-chart)
+- [3 References](#references)
+  * [ZMQ](#zmq)
+
+# 1 Functional Requirement
+## 1.1 Supported operations
+ - Should support following operations.
```

Comment:
>operations [](http://example.com/codeflow?start=28&length=10)

Is it possible to add parameters in below operation list?

Reply:
Fixed, design doc updated.